packages/ckeditor5-engine/tests/view/matcher.js

```diff
@@ -176,6 +317,25 @@ describe( 'Matcher', () => {
 			expect( new Matcher( { classes: 'baz' } ).match( el1 ) ).to.be.null;
 		} );
 
+		it( 'should match element class names using an array', () => {
+			const pattern = { classes: [ 'foobar', 'foobaz' ] };
+			const matcher = new Matcher( pattern );
+			const el1 = new Element( document, 'p', { class: 'foobar foobaz' } );
```

Comment:
Maybe just `foo` and `bar`? Nothing important thou, it would be just a bit more readable when reading expected results.